getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -224,14 +201,9 @@ you should be aware of when diving into C# in Godot, but if in doubt, also
 take a look over the official
 `issue tracker for Mono issues <https://github.com/godotengine/godot/labels/topic%3Amono>`_.
 
-- As explained above, the C# project isn't always kept in sync automatically
-  when things are deleted, renamed or moved in Godot
-  (`#12917 <https://github.com/godotengine/godot/issues/12917>`_).
 - Writing editor plugins is possible, but it is currently quite convoluted.
 - State is currently not saved and restored when hot-reloading,
   with the exception of exported variables.
-- Exporting Mono projects is supported for desktop platforms
-  (Linux, Windows and macOS), Android, HTML5, and iOS. UWP is not currently supported.
   (`#20271 <https://github.com/godotengine/godot/issues/20271>`_).
```

Comment:
You forgot to remove this line.

Reply:
@akien-mga Good catch :slightly_smiling_face: Fixed.